File: Java/Iniciante/1006 - Média 2..java
/*
 * Autor: Pierre Vieira
 * Data de submissão:
 */
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner teclado = new Scanner(System.in);
        double a, b, c;
        a = teclado.nextDouble();
        b = teclado.nextDouble();
        c = teclado.nextDouble();
        System.out.printf("MEDIA = %.1f\n", (2*a+3*b+5*c)/10);
    }
}

File: C#/Iniciante/1006 - Média 2.cs
/*
* Autor: Pierre Vieira
* Data de submissão: 13/05/2020 12:11:10
*/
using System;
using System.Globalization;

namespace URI
{
    class URI
    {
        static void Main(string[] args)
        {
            double n1, n2, n3;
            n1 = 2 * double.Parse(Console.ReadLine());
            n2 = 3 * double.Parse(Console.ReadLine());
            n3 = 5 * double.Parse(Console.ReadLine());
            Console.WriteLine("MEDIA = "+((n1 + n2 + n3)/10).ToString("F1", CultureInfo.InvariantCulture));
        }
    }
}
